addons/stock/models/stock_orderpoint.py

```diff
@@ -257,8 +262,15 @@ def _compute_qty_to_order(self):
                 remainder = orderpoint.qty_multiple > 0 and qty_to_order % orderpoint.qty_multiple or 0.0
                 if float_compare(remainder, 0.0, precision_rounding=rounding) > 0:
                     qty_to_order += orderpoint.qty_multiple - remainder
+
             orderpoint.qty_to_order = qty_to_order
 
+    def _get_min_qty_to_order(self):
+        """ Calculates the minimum quantity that can be ordered according to the PO UoM or BoM
+        """
+        self.ensure_one()
+        return False
```

Comment:
return a False for a number is quite not intuitive (it works, but is weird), maybe return 0 ?

Reply:
Yes, `False` does not make much sense. I would return `None` instead

Thanks for all the feedback ! :-)